sos/report/plugins/__init__.py

```diff
@@ -2011,7 +2011,7 @@ def _add_cmd_output(self, **kwargs):
         if self.get_option('all_logs') or kwargs['sizelimit'] == 0:
             kwargs['to_file'] = True
         soscmd = SoSCommand(**kwargs)
-        self._log_debug("packed command: " + soscmd.__str__())
+        self._log_debug("packed command: " + str(soscmd))
```

Comment:
We might move to f-strings here already, but that would require bit bigger change here, I think (since f-strings use `format()` method, not currently called `__str_` or `str()` one).

Reply:
I think f-string should work here:
```
>>> s=1
>>> o=f"test: {str(s)}"
>>> print(o)
```

Reply:
Switched to f-strings for the two changes in this file.